src/custom-types.json

```diff
@@ -878,7 +902,8 @@
     "compressible": false
   },
   "video/webm": {
-    "compressible": false
+    "compressible": false,
+    "extensions" : ["webm"]
```

Comment:
Please make sure to add a `sources` array here with links to the primary sources for the association of these extensions to this mime type.